feat: add expiry date to strapi config

luke-z opened the pull request:
<!--- Provide a general summary of your changes in the title above -->
As this is my first open source pr, I hope I did everything well :)

## Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)


## Description
Currently the strapi token is saved as a session cookie which expires as soon as the browser is closed.

Back in v0, there was a [config option](https://strapi-v0.nuxtjs.org/options#expires) which allowed to set the expires property.
With this pull request I have added such a feature.

## Checklist:
<!--- Put an `x` in all the boxes that apply. -->
<!--- If your change requires a documentation PR, please link it appropriately -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [-] I have added tests to cover my changes (if not applicable, please state why)

benjamincanac commented:
Hi @luke-z,

First of all, thank you for this PR!

Back in the `v0` of the module, the `expires` key was here to tell the module to use `sessionStorage` instead of `cookies`. With this new module, we're only using cookies and the default is indeed set as `session` as stated in the Nuxt 3 [useCookie expires](https://v3.nuxtjs.org/docs/usage/cookies#maxage--expires) documentation, which will behave differently depending on the browser.

To change this behaviour, I would rather only have a `cookie` key in the options, typed by `CookieOptions` to configure `useCookie` composable.

luke-z commented:
Hello @benjamincanac 

Thank you for the feedback!
It makes more sense to have all the options available.

I changed the config from expires to cookie as you suggested and updated the docs.

benjamincanac commented on src/types/index.ts:
```diff
@@ -1,3 +1,4 @@
+import { CookieOptions } from 'nuxt3/dist/app/composables/cookie'
```
@danielroe To confirm with you, is this the right way to get this type?

luke-z replied:
I just tried using the type without any import, this results in the build error 

`error TS4033: Property 'cookie' of exported interface has or is using private name 'CookieOptions'.`

benjamincanac replied:
With `nuxt3` import it resulted into a build error, I've updated it with the `#app` alias but we still have a warning on build:

```
WARN  Inlining implicit external #app/composables/cookie                                                                                                                    

'#app/composables/cookie' is imported by src/types/index.ts, but could not be resolved – treating it as an external dependency
```

I'm gonna wait for Daniel validation on this.

luke-z replied:
I put #app in the build.config.ts, which seems to resolve the problem (I am not familiar with rollup)

`externals: ['@nuxt/kit', '@nuxt/schema', 'vue', '#app']`

benjamincanac replied:
Thanks a lot @danielroe, it works perfectly!
